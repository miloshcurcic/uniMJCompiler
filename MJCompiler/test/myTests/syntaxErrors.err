ERROR 20:22:45,495 - Line 5: Syntax error detected
ERROR 20:22:45,498 - Line 5: Syntax error detected
ERROR 20:22:45,501 - Line 7: Syntax error detected
ERROR 20:22:45,501 - Line 8: Syntax error detected
ERROR 20:22:45,502 - Line 8: Syntax error detected
ERROR 20:22:45,504 - Line 14: Syntax error detected
ERROR 20:22:45,506 - Line 15: Syntax error detected
ERROR 20:22:45,507 - Line 17: Syntax error detected
ERROR 20:22:45,508 - Line 20: Syntax error detected
ERROR 20:22:45,509 - Line 20: Syntax error detected
ERROR 20:22:45,510 - Line 21: Syntax error detected
ERROR 20:22:45,512 - Line 25: Syntax error detected
ERROR 20:22:45,513 - Line 29: Syntax error detected
