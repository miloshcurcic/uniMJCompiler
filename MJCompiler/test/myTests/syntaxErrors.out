INFO  20:19:38,196 - ===================================
INFO  20:19:38,199 - Compiling source file: H:\Fakultet\Semestar 7\PP\MJCompiler\test\test304.mj
INFO  20:19:38,200 - ===================================
ERROR 20:19:38,285 - Line 5: Syntax error detected
INFO  20:19:38,286 - Line 5: Successfully recovered from variable declaration syntax error!
ERROR 20:19:38,287 - Line 5: Syntax error detected
INFO  20:19:38,287 - Line 5: Successfully recovered from variable declaration syntax error!
ERROR 20:19:38,289 - Line 7: Syntax error detected
INFO  20:19:38,290 - Line 7: Successfully recovered from class extension syntax error!
ERROR 20:19:38,291 - Line 8: Syntax error detected
INFO  20:19:38,291 - Line 8: Successfully recovered from variable declaration syntax error!
ERROR 20:19:38,296 - Line 8: Syntax error detected
INFO  20:19:38,296 - Line 8: Successfully recovered from variable declaration syntax error!
ERROR 20:19:38,300 - Line 14: Syntax error detected
INFO  20:19:38,301 - Line 14: Successfully recovered from assignment syntax error!
ERROR 20:19:38,302 - Line 15: Syntax error detected
INFO  20:19:38,302 - Line 15: Successfully recovered from assignment syntax error!
ERROR 20:19:38,303 - Line 17: Syntax error detected
INFO  20:19:38,303 - Line 17: Successfully recovered from assignment syntax error!
ERROR 20:19:38,304 - Line 20: Syntax error detected
INFO  20:19:38,305 - Line 20: Successfully recovered from formal parameter declaration syntax error!
ERROR 20:19:38,305 - Line 20: Syntax error detected
INFO  20:19:38,305 - Line 20: Successfully recovered from formal parameter declaration syntax error!
ERROR 20:19:38,306 - Line 21: Syntax error detected
INFO  20:19:38,307 - Line 21: Successfully recovered from if condition syntax error!
ERROR 20:19:38,309 - Line 25: Syntax error detected
INFO  20:19:38,309 - Line 25: Successfully recovered from if condition syntax error!
ERROR 20:19:38,311 - Line 29: Syntax error detected
INFO  20:19:38,311 - Line 29: Successfully recovered from if condition syntax error!
INFO  20:19:38,312 - ===================================
INFO  20:19:38,312 - One or more lexical or syntax errors detected.
INFO  20:19:38,312 - ===================================
